src/host/ledger.h

```diff
@@ -1423,21 +1434,15 @@ namespace asynchost
               job->ledger = this;
               job->from_idx = from_idx;
               job->to_idx = to_idx;
-              job->result_cb = [this,
-                                from_idx = from_idx,
-                                to_idx = to_idx,
-                                purpose = purpose,
-                                strict = strict,
-                                max_entries_size =
-                                  max_entries_size](auto&& entry, int status) {
-                // NB: Even if status is cancelled (and entry is empty), we
-                // want to write this result back to the enclave
-                write_ledger_get_range_response(
-                  from_idx,
-                  to_idx,
-                  read_entries(from_idx, to_idx, strict, max_entries_size),
```

Comment:
This is a funny bug and hopefully a big saving - we were previously calling `read_entries` twice! Once in the `get_async` callback, and again in the `get_async_complete` callback! The latter is now correctly just forwarding the results from the earlier call.